src/support.njk

```diff
@@ -1,27 +1,114 @@
+<div class="flex px-6 -mt-32 mx-auto text-gray-400 md:max-w-screen-lg md:-mt-40 mb-8 md:mb-16">
+  <p class="lead-p max-w-screen-lg m-auto text-center md:max-w-xl md:px-4">
+    Search through our extensive range of resources, knowledge base, or reach out to our dedicated support team by
+    submitting a ticket.
+  </p>
+</div>
+<div class="text-left max-w-full ff-bg-light">
+  <div class="w-full pt-10 md:pt-8 px-0 ff-prose">
+    <div class="text-center md:mt-10 mb-8">
+      <h2>Find answers instantly</h2>
     </div>
+    <div class="max-w-lg m-auto px-6 md:px-0 md:mb-10">
+      <div id="algolia-search" class="border rounded"></div>
   </div>
-</div>
+    <div class="container m-auto max-w-screen-lg ">
+      <div class="grid px-6 grid-cols-1 md:grid-cols-2 gap-6 md:gap-x-12 gap-y-12 mb-16">
+        <div class="md:flex md:flex-col md:justify-between text-left self-start md:h-full">
+          <div class="prose">
+            <h3 class="flex items-center gap-4">
+              <div class="w-16 mt-6">
+                {% image "./images/pictograms/community_red.png", "Graphics depicting low-code programming.", [64] %}
+              </div>
+              <div class="flex flex-col leading-8 mt-6">
+                Ask the community
+              </div>
+            </h3>
+            <p class="mb-0">
+              Explore our forum to find answers and solutions. Others may have already shared similar issues, providing
+              valuable insights that could address your concerns.
+            </p>
+          </div>
+          <div
+            class="container text-center max-w-4xl flex flex-col md:items-start mt-6">
+            <a class="ff-btn ff-btn--primary flex flex-col" href="https://community.flowforge.com/">
+              <span class="text-base uppercase no-underline md:px-4">
+                COMMUNITY FORUM
+              </span>
+            </a>
+          </div>
+        </div>
+        <div class="md:flex md:flex-col md:justify-between text-left self-start md:h-full">
+          <div class="prose">
+            <h3 class="flex items-center gap-4">
+              <div class="w-16 mt-6">
+                {% image "./images/pictograms/speech_red.png", "Graphics depicting low-code programming.", [64] %}
+              </div>
+              <div class="flex flex-col leading-8 mt-6">
+                Reach out via our chat
+              </div>
+            </h3>
+            <p class="mb-0">
+              Get real-time assistance and quick answers. Chat with us for prompt support and solutions to your queries.
+            </p>
+          </div>
+          <div class="container text-center max-w-4xl flex flex-col md:items-start mt-6">
+            <button onclick="window.HubSpotConversations.widget.open()" class="ff-btn ff-btn--primary flex flex-col">
+              <span class="text-base uppercase no-underline md:px-4">
+                open chat box
```

Comment:
Nit: COMMUNITY FORUM is in all caps too

```suggestion
                OPEN CHAT BOX
```

Reply:
The "uppercase" class takes care of that ;) 
Will change it anyways just to tidy it up